scrapers/LegalPorno.yml

```diff
@@ -0,0 +1,31 @@
+name: "LegalPorno"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - legalporno.com/watch/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $description: //dl[@class="dl-horizontal scene-description__column"]
+    scene:
+      Title:
+        selector: //h1[@class="watchpage-title"]//text()
+        concat: " "
+      Date:
+        selector: //span[@title="Release date"]/a/text()
+        parseDate: 2006-01-02
+      Details: $description/div[3]/dd/text()
+      Performers:
+        Name: $description/div[1]/dd/a[position() mod 2 = 1]/text()
```

Comment:
performers are not always every two links eg https://www.legalporno.com/watch/34413/veronica_leal_roughed_up_double_penetrated_by_four_men_sz2073
```
Name: $description/div[1]/dd/a[contains(@href,'legalporno.com/model')]/text()
```
seems to work

Reply:
Fixed!